data-prepper-plugins/saas-source-plugins/crowdstrike-source/src/main/java/org/opensearch/dataprepper/plugins/source/crowdstrike/models/CrowdStrikeApiResponse.java

```diff
@@ -0,0 +1,34 @@
+package org.opensearch.dataprepper.plugins.source.crowdstrike.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.util.CollectionUtils;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents the response returned from a CrowdStrike API call.
+ */
+@Getter @Setter
+ public class CrowdStrikeApiResponse {
```

Comment:
nit: can we rename this as CrowdstrikeIntelApiResponse for disambiguation purposes.. since there are other APIs we could be integrating in the future